tests/api/v1/generators/post.js

```diff
@@ -58,6 +58,7 @@ describe('tests/api/v1/generators/post.js >', () => {
 
       expect(res.body.apiLinks).to.be.an('Array');
       expect(res.body.name).to.include(generator.name);
+      expect(res.body).to.have.property('intervalSecs', 60);
```

Comment:
Should also have a test case where the value is actually set.

Reply:
yes, done